src/sys.d.ts

```diff
@@ -0,0 +1,1352 @@
+export type JsObj = Obj | number | string | boolean | ((...args: any) => any)
+
+export class Obj {
+  equals(that: JsObj | null): boolean
```

Comment:
Whenever the argument would be nullable the translatin should be `that?: JsObj`

Reply:
What about function arguments that are nullable but default to something other than null? Take something like this in Fantom:

```Fantom
Void setSomething(Bool? val := true) { someVar = val }

// first sets to null, then to true
setSomething(null)
setSomething
```

Which would look something like this in JS, using undefined instead of null:
```JS
setSomething(val=true) { someVar = val }

// these two are the same - both set to true
setSomething(undefined);
setSomething();
```

Is there a way to prevent this incorrect translation without using null?

Reply:
This would translate to:
```typescript
setSomething(val: boolean | undefined = true) { ... }
```